sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -4022,6 +4023,46 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     }
   }
 
+  test("SPARK-33474: Support TypeConstructed partition spec value") {
```

Comment:
let's move the test to `SQLInsertTestSuite`

Reply:
> let's move the test to `SQLInsertTestSuite`

Done